spec/lib/bank_account/account_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'bank_account'
+
+describe Bank::Account do
+  let(:account) {Bank::Account.new("check_it", 1000)}
+
+  describe "attributes" do
+    it "responds to :balance" do
+      expect(account).to respond_to :balance
+    end
+
+    it "responds to :id" do
+      expect(account).to respond_to :id
+    end
```

Comment:
Which test is better form to write? The "responds to" or the name of an example account?